test/testHelpers.hpp

```diff
@@ -26,6 +26,19 @@ typedef unsigned char  uchar;
 typedef unsigned int   uint;
 typedef unsigned short ushort;
 
+std::string readLine(std::ifstream &file)
+{
+    std::string result = "";
+    // Using a for loop to read the next non empty line
+    for (std::string line; std::getline(file, line);) {
+        result += line;
+        if (result != "") break;
+    }
+    // If no file has been found, throw an exception
+    if (result == "") throw std::runtime_error("Test file might not be per format, please check.");
```

Comment:
Can you change the name to readNextNonEmptyLine and also update the error message?